H2MLauncher.UI/ViewModels/ServerBrowserViewModel.cs

```diff
@@ -63,6 +65,10 @@ public partial class ServerBrowserViewModel : ObservableObject
     [NotifyPropertyChangedFor(nameof(IsRecentsSelected))]
     private IServerTabViewModel _selectedTab;
 
+    [ObservableProperty]
+    private ServerViewModel? _lastServer = null;
+    private SecureString? _lastServerPassword = null;
```

Comment:
The thing about making the password thing secure, is that the moment it is entered in the textbox and send as command to the application it is visible in plain text in the console.

Reply:
Yeah true, doesn't help much. If any, makes it a little harder to get it, but then, it is not really super secret. Do you want me to remove it or is it fine to leave anyways?